services/data/BookHelper.cs

```diff
@@ -121,7 +124,13 @@ static Book GetUserInputAsBook()
             Console.WriteLine("\nPlease enter the publish date: ");
             int publishDate = int.Parse(Console.ReadLine());
 
-            return new Book(pid, title, author, publishDate);
+            Book book = new Book();
```

Comment:
Use class instantiation to set properties instead of setting props one at a time

```
var book = new Book {
  Id = pid,
  Title = title,
 ...
};
```